packages/database/src/migrations/20201009035426-AddImmutableTableTrigger-modifies-schema.js

```diff
@@ -31,7 +31,7 @@ exports.up = function (db) {
 
 exports.down = function (db) {
   return db.runSql(`
-    DROP FUNCTION IF EXISTS immutable_table();
+    DROP FUNCTION IF EXISTS immutable_table() CASCADE;
```

Comment:
Unrelated, just found when I was trying to migrate down to replay the migrations